### DISCUSSION TITLE
Fine-tuned large-v3 model does not transcribe non-English such as Chinese, Malay

### DISCUSSION THREAD

**Discussion by toanhuynhnguyen** [SYSTRAN/faster-whisper#1072] [Q&A]
Based on this guide https://huggingface.co/blog/fine-tune-whisper, I tried to fine-tune "small" and "large-v3" models.
- The fine-tuned "small" model works normally.
- But the fine-tuned "large-v3" model works poorly on non-English audio files such as Chinese audio files, it auto-translates Chinese to English though I specified transcribing in Chinese not translating.
Have you faced this issue, and can give me advice? Thank you so much.

**Comment by techspecshq** [comment]
> Based on this guide https://huggingface.co/blog/fine-tune-whisper, I tried to fine-tune "small" and "large-v3" models.
> 
> * The fine-tuned "small" model works normally.
> * But the fine-tuned "large-v3" model works poorly on non-English audio files such as Chinese audio files, it auto-translates Chinese to English though I specified transcribing in Chinese not translating.
>   Have you faced this issue, and can give me advice? Thank you so much.

Can you upload the audio file you used so that I can try it out on my end
